.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.0
```

Comment:
Alternatively I could put the latest patch version here, `2.4.5`.

Reply:
what about having the last Ruby version: 2.6?